pyrate/core/gamma.py

```diff
@@ -38,6 +38,20 @@
 GAMMA_DATUM = 'ellipsoid_name'
 GAMMA_FREQUENCY = 'radar_frequency'
 GAMMA_INCIDENCE = 'incidence_angle'
+GAMMA_HEADING = 'heading'
+GAMMA_RANGE_PIX = 'range_pixel_spacing'
+GAMMA_RANGE_N = 'range_samples'
+GAMMA_AZIMUTH_PIX = 'azimuth_pixel_spacing'
+GAMMA_AZIMUTH_N = 'azimuth_lines'
+GAMMA_AZIMUTH_LOOKS = 'azimuth_looks'
+GAMMA_PRF = 'prf'
+GAMMA_NEAR_RANGE = 'near_range_slc'
+GAMMA_SAR_EARTH = 'sar_to_earth_center'
+GAMMA_SEMI_MAJOR_AXIS = 'earth_semi_major_axis'
+GAMMA_SEMI_MINOR_AXIS = 'earth_semi_minor_axis'
+# to do add option to use initial baseline if precision baseline was not calculated in GAMMA
```

Comment:
as a tip, use "TODO" instead of "to do". TODO is recognised as a flag in `pylint` code checking and will be collated in to a list of TODO's

Reply:
see https://github.com/GeoscienceAustralia/PyRate/pull/302/commits/fa2a65850f4aec5bfc26712c9339f19b6edc51ee